src/rascal/math/spherical_harmonics.cc

```diff
@@ -134,7 +134,7 @@ void SphericalHarmonics::compute_assoc_legendre_polynom(double cos_theta) {
 
 void SphericalHarmonics::calc(
     const Eigen::Ref<const Eigen::Vector3d> & direction,
-    bool calculate_derivatives) {
+    bool calculate_derivatives, bool conjugate) {
```

Comment:
I also thought a lot about it and I thought it might be. I sense this real/complex thing is going to haunt us again, and this (as well as the realconjugate code on the python side) might be helpful to have around.

Reply:
fair enough!